pom.xml

```diff
@@ -59,7 +59,7 @@
     <!-- We skip sources jar generation as we do it with the assembly plugin to have greater control over the content -->
     <source.skip>true</source.skip>
     <kotlin.version>1.3.0</kotlin.version>
-    <testcontainers.version>1.12.0</testcontainers.version>
+    <testcontainers.version>1.12.4</testcontainers.version>
```

Comment:
We cannot change the version without a works-with CQ - can you revert this and make a separate PR for which we will create the works-with CQ separately ?

Reply:
Submitted the CQ for testcontainers update here: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21539